app/src/main/res/layout/onboarding_slide.xml

```diff
@@ -43,7 +43,7 @@
         android:layout_marginBottom="16dp"
         android:fontFamily="sans-serif-medium"
         android:gravity="center_horizontal"
-        android:maxLines="1"
+        android:maxLines="2"
```

Comment:
Same previous comment applies to this code too.

Remove maxLines attribute.

Also, please re-generate screenshots and update PR description just to be sure

Reply:
Noted.